src/main/java/org/fogbowcloud/saps/engine/scheduler/restlet/resource/ProcessedTasksResource.java

```diff
@@ -0,0 +1,122 @@
+package org.fogbowcloud.saps.engine.scheduler.restlet.resource;
+
+import java.util.Date;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.fogbowcloud.saps.engine.core.model.ImageTask;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.restlet.data.Form;
+import org.restlet.data.MediaType;
+import org.restlet.data.Status;
+import org.restlet.representation.Representation;
+import org.restlet.representation.StringRepresentation;
+import org.restlet.resource.Post;
+import org.restlet.resource.ResourceException;
+
+/**
+ * Responsable for retrieve information about already processed tasks.
+ */
+public class ProcessedTasksResource extends BaseResource {
+
+	private static final Logger LOGGER = Logger.getLogger(ProcessedTasksResource.class);
+
+	private static final String LOWER_LEFT = "lowerLeft";
+	private static final String UPPER_RIGHT = "upperRight";
+	private static final String PROCESSING_INIT_DATE = "initialDate";
+	private static final String PROCESSING_FINAL_DATE = "finalDate";
+	private static final String PROCESSING_INPUT_GATHERING_TAG = "inputGatheringTag";
+	private static final String PROCESSING_INPUT_PREPROCESSING_TAG = "inputPreprocessingTag";
+	private static final String PROCESSING_ALGORITHM_EXECUTION_TAG = "algorithmExecutionTag";
+
+	public ProcessedTasksResource() {
+		super();
+	}
+
+	/**
+	 * Returns all processed tasks that matches the execution parameters passed
+	 * in {@code representation}.
+	 *
+	 * @param representation Entity containing execution parameters.
+	 * @return List of processed tasks.
+	 */
+	@Post
```

Comment:
this method seems too large. How about creating some private methods to split this big one?

Reply:
In my opinion this huge parse of specified `representation` should be done in other method, as well as the creation of response object. But this method I created is a "copy" of RegionResource#getProcessedImagesInInterval without authentication, because a user of a instance doesn't exists to other instance.

But I will do as you suggested and split this method, making the parse in a easier and reusable way.